azure-mgmt-locationbasedservices/azure/mgmt/locationbasedservices/client.py

```diff
@@ -0,0 +1,85 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from msrest.service_client import ServiceClient
+from msrest import Serializer, Deserializer
+from msrestazure import AzureConfiguration
+from .version import VERSION
+from .operations.accounts_operations import AccountsOperations
+from . import models
+
+
+class ClientConfiguration(AzureConfiguration):
+    """Configuration for Client
+    Note that all parameters used to create this instance are saved as instance
+    attributes.
+
+    :param credentials: Credentials needed for the client to connect to Azure.
+    :type credentials: :mod:`A msrestazure Credentials
+     object<msrestazure.azure_active_directory>`
+    :param subscription_id: Subscription credentials which uniquely identify
+     Microsoft Azure subscription. The subscription ID forms part of the URI
+     for every service call.
+    :type subscription_id: str
+    :param str base_url: Service URL
+    """
+
+    def __init__(
+            self, credentials, subscription_id, base_url=None):
+
+        if credentials is None:
+            raise ValueError("Parameter 'credentials' must not be None.")
+        if subscription_id is None:
+            raise ValueError("Parameter 'subscription_id' must not be None.")
+        if not base_url:
+            base_url = 'https://management.azure.com'
+
+        super(ClientConfiguration, self).__init__(base_url)
+
+        self.add_user_agent('azure-mgmt-locationbasedservices/{}'.format(VERSION))
+        self.add_user_agent('Azure-SDK-For-Python')
+
+        self.credentials = credentials
+        self.subscription_id = subscription_id
+
+
+class Client(object):
```

Comment:
didn't realize, but that's too generic and should be "LocationBasedServicesClient" at least
Why is that?
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/locationbasedservices/resource-manager/Microsoft.LocationBasedServices/preview/2017-01-01-preview/locationbasedservices.json#L8

Reply:
It must have been an oversight on our end.
Do you know if the rest-api-specs team should have caught this before it was merged?

Reply:
Yes, they should have reviewed that :(

Reply:
Please do another Swagger PR to fix this. Assign it to me (or tag me if you don't have assign permission) so it will be faster. Sorry about that :(

Reply:
Got it. Do you recommend PR with a new (version), or update the current?

Reply:
update the current Swagger